### DISCUSSION TITLE
workaround for getting wavespectra to work in a pip environment if you do not have a fortran compiler

### DISCUSSION THREAD

**Discussion by RubendeBruin** [wavespectra/wavespectra#115] [Show and tell]
TLDR;
install using conda in a separate environment, then copy the package folder to your pip environment

Why:
If you do not have a fortran compiler, then you can not build the package yourself.

Step1:
Get miniconda3 (https://docs.anaconda.com/free/miniconda/index.html)

Step2:
make a miniconda environment for the same python version as you have in your other python environment.
`conda create -n temp python=3.11`
and activate it
`activate temp`

Step3:
install mamba
`conda install -c conda-forge mamba`

Step4:
install wavespecta using mamba
`mamba install -c conda-forge wavespectra`

Step5:
Locate the installed folder

![image](https://github.com/wavespectra/wavespectra/assets/34062862/d2c80ea7-b80b-4913-b0a9-e4b68843d0b5)

Step6:
Copy it to your main python environment.
That means: copy the whole "wavespectra" folder from yourcondadist/libs/site-packages/ to yourmainpython/libs/site-packages

Step7:
Delete your conda stuff, we will not longer need it

Step8:
Go to your main python environment.

Install all wave-spectra requirementes:

`pip install click cmocean dask matplotlib numba pandas python-dateutil pyyaml sortedcontainers scipy toolz typing_extensions xarray`

Step9:
Keep your fingers crossed and give it a try:
![image](https://github.com/wavespectra/wavespectra/assets/34062862/50f53b8b-7178-4ab9-b7a2-d029633aa548)

**Comment by rafa-guedes** [comment]
Closing this as fortran is no longer required in wavespectra
